keps/sig-network/2433-topology-aware-hints/README.md

```diff
@@ -307,9 +309,14 @@ had not yet propagated to all of them.
 ### EndpointSlice Controller
 
 When the `TopologyAwareHints` feature gate is enabled and the annotation is set
-to `Auto` for a Service, the EndpointSlice controller will add hints to
-EndpointSlices. These hints will indicate where an endpoint should be consumed
-by proxy implementations to enable topology aware routing.
+to `Auto` or `SameZone` for a Service, the EndpointSlice controller will add
+hints to EndpointSlices. These hints will indicate where an endpoint should be
+consumed by proxy implementations to enable topology aware routing.
+
+Each heuristic (`Auto` or `SameZone`) will have different logic to determine how
+hints should be allocated. These will be described below:
+
+### Auto Heuristic
```

Comment:
To be precise: "auto" isn't a heuristic - it's a meta-heuristic.  It means "choose for me".  We didn't give the existing heuristic a distinct name because we didn't really have anything to compare it to.

If we were to go this path, we would probably want to give it a name like "CoreProportional" or something